generators/java/index.js

```diff
@@ -0,0 +1,155 @@
+const Generator = require('yeoman-generator');
+const keyPair = require('keypair');
+const path = require('path');
+const colors = require('colors');
+const axios = require('axios');
+const config = require('../lib/config');
+
+const BASE_JAVA = 'src/main/java';
+const BASE_RESOURCES = 'src/main/resources';
+const DEFAULT_SDK_VERSION = '1.0.50'; // just in case if the search query failed
+const MAVEN_SEARCH_QUERY = 'https://search.maven.org/solrsearch/select?q=g:com.symphony.platformsolutions+AND+a:symphony-api-client-java';
+
+module.exports = class extends Generator {
```

Comment:
could we add what is the role of this class?

Reply:
Done